lib/Rex/Pkg/Base.pm

```diff
@@ -66,11 +66,22 @@ sub update {
   my $env     = $option->{'env'}     || ();
 
   Rex::Logger::debug( "Installing $pkg" . ( $version ? "-$version" : "" ) );
-  my $cmd = sprintf $self->{commands}->{install}, $pkg;
+  my $cmd;
+  if ( ( $pkg =~ /\*/ )
+    && defined( $self->{commands}->{install_glob} ) )
```

Comment:
Yup!

I am wondering if we want to check for * by default and then processing packages. For example apt does not mind globs, but * also needs escaped for that.

Reply:
If I understand you correctly, yes, it might be possible and useful to always escape the contents of `$pkg`, and not just in the cases when we think it would need escaping.

If you would like to work on that on this PR, please mark it as draft, hack on it, and mark it "ready for review" again when you are done, and tests pass.

Alternatively, it might be just as good to do that on a separate follow-up PR (or to open an issue about it as a reminder).